Example/Tests/SmokeAPITests/SmokeAPITests.swift

```diff
@@ -0,0 +1,47 @@
+import XCTest
+import class Foundation.Bundle
+
+final class SmokeAPITests: XCTestCase {
+    func testExample() throws {
+        // This is an example of a functional test case.
+        // Use XCTAssert and related functions to verify your tests produce the correct
+        // results.
+
+        // Some of the APIs that we use below are available in macOS 10.13 and above.
+        guard #available(macOS 10.13, *) else {
```

Comment:
Is there a reason the tests are just for macOS?

Reply:
Deleted my test case which were generated automatically by system.